modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/Paths.kt.mustache

```diff
@@ -33,6 +19,7 @@ object Paths {
      * {{#unescapedNotes}}{{.}}{{/unescapedNotes}}
      {{#allParams}}* @param {{paramName}} {{description}} {{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
      {{/allParams}}*/
+    @KtorExperimentalLocationsAPI
```

Comment:
Generating the samples code, I get a ton of messages from the *Api.kt files:

```
w: /Users/jim/projects/openapi-generator/samples/server/petstore/kotlin-server/ktor/src/main/kotlin/org/openapitools/server/apis/UserApi.kt: (90, 40): This declaration is experimental and its usage should be marked with '@io.ktor.locations.KtorExperimentalLocationsAPI' or '@UseExperimental(io.ktor.locations.KtorExperimentalLocationsAPI::class)'
```

Do you see this when building locally? I wonder if we need this annotation in both places.

Reply:
thanks for the review
i updated the PR to fix all compilation warnings
please let me know if it's ok now